apps/vr-tests/src/stories/SpinButton.stories.tsx

```diff
@@ -48,17 +50,67 @@ storiesOf('SpinButton', module)
     'With icon',
     () => (
       <Fabric>
-        <SpinButton {...props} iconProps={{ iconName: 'IncreaseIndentLegacy' }} />
+        <SpinButton {...props} iconProps={iconProps} />
       </Fabric>
     ),
     { rtl: true }
-  )
+  );
+
+// The stories for label placement are separate since they don't need to include hover/click states
+storiesOf('SpinButton - Label Placement', module)
+  .addDecorator(FabricDecorator)
+  .addDecorator(story => (
+    <Screener steps={new Screener.Steps().snapshot('default', { cropTo: '.testWrapper' }).end()}>
+      {story()}
+    </Screener>
+  ))
   .addStory(
     'Label at end',
     () => (
       <Fabric>
-        <SpinButton {...props} labelPosition={Position.end} />
+        <SpinButton {...props} styles={styles} labelPosition={Position.end} />
       </Fabric>
     ),
     { rtl: true }
-  );
+  )
+  .addStory(
+    'Label at end with icon',
+    () => (
+      <Fabric>
+        <SpinButton {...props} styles={styles} labelPosition={Position.end} iconProps={iconProps} />
+      </Fabric>
+    ),
+    { rtl: true }
+  )
+  .addStory('Label on top', () => (
```

Comment:
This is what I meant by vertically aligned label - should we add examples to the Spinbutton page to display the fix to this use case?

Reply:
I like the idea of adding an example demonstrating label placement, but I don't think we need to specifically demonstrate the fix to this issue. We don't specifically demonstrate the alignment with other form component examples, it's expected to "just work" (but it's good to have a test to verify it continues working).